Jenkinsfile

```diff
@@ -19,46 +19,138 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-@Library(value="pipeline-lib@corci-734") _
+// The @library line needs to be edited in a PR for adding a test.
+// This needs be commented out before deleting the branch used for a PR.
+@Library(value="pipeline-lib@jem-corci-749") _
```

Comment:
My vote is to land with this included so that the tests pass before we land and then push another PR to revert it back to master and land it once it passes again.